Jmeter/UserStory3.jmx

```diff
@@ -0,0 +1,1407 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.6.3">
```

Comment:
so your report is not what i would like to see, 
1/ did you generate this report https://haquemousume.medium.com/learn-to-generate-and-analyse-html-intuitive-reports-in-jmeter-574dbe1fca72

also a video on it https://www.youtube.com/watch?v=cGratqVrQTA&ab_channel=HelpingTesters 

2/ where is your analysis on the performance test, were the results for request times good or bad, i need a little report on this! (stating if the response times meet the expected see acceptance criteria)